### PR TITLE
Completed prep-css-layout-classes.  However, the webpage is not looki…

### PR DESCRIPTION
**Pull request by edtan094**
…ng the way I want it too for some reason.  Edwin Tan
prep-css-layout-classes screenshot

### CODE REVIEW

File: prep-css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  </style>
+</head>
+<body>
+
+<!-- Complete the HTML here -->
+<h3>Indian Jones</h3>
+<div class="container">
+</div>
+  <div class="row">
+  </div>
```
Comment (thebearingedge) [inline prep-css-layout-classes/index.html]: Hi Edwin, try moving your columns into here

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-layout-classes/index.html (hunk 1 above)
Comment (edtan094) [reply prep-css-layout-classes/index.html]: prep-css-layout-classes screenshot

File: prep-css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      padding-right: 8px;
+    }
+  </style>
+</head>
+<body>
+
+<!-- Complete the HTML here -->
+<h3>Indian Jones</h3>
+<div class="container">
+</div>
```
Comment (thebearingedge) [inline prep-css-layout-classes/index.html]: Ok, much better. Now you can put the row in here so that the page is centered

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-layout-classes/index.html (hunk 2 above)
Comment (edtan094) [reply prep-css-layout-classes/index.html]: prep-css-layout-classes screenshot
Looks so much better with just two changes

**Review by thebearingedge — APPROVED** [review]
👏 👏 👏

**Pull request merged**